iron-selectable.html

```diff
@@ -336,10 +346,19 @@
       if (this.selectedAttribute) {
         this.toggleAttribute(this.selectedAttribute, isSelected, item);
       }
+      if (this.selectedPropertyValue) {
+        item[this.selectedAttribute] = this.selectedPropertyValue;
+      }
       this._selectionChange();
       this.fire('iron-' + (isSelected ? 'select' : 'deselect'), {item: item});
     },
 
+    _updatePropertyValue: function(selectedItem, selectedAttribute, propertyValue) {
+      if (selectedAttribute && propertyValue) {
+        selectedItem[selectedAttribute] = propertyValue;
```

Comment:
Yes but `_applySelection` would only be called on change of the selected item, if you have an `app-route` inside your view binding to the `route` property and then handling other views, how would it get updates for when the page changes?